### ISSUE TITLE
基于ptuning训练完成，新老模型都进行了加载，但是只有新的

### ISSUE THREAD

**Issue by zhengyao123** [imClumsyPanda/langchain-ChatGLM#280]
licitly passing a `revision` is encouraged when loading a model with custom code to ensure no malicious code has been contributed in a newer revision.
Explicitly passing a `revision` is encouraged when loading a configuration with custom code to ensure no malicious code has been contributed in a newer revision.
Explicitly passing a `revision` is encouraged when loading a model with custom code to ensure no malicious code has been contributed in a newer revision.
Loading checkpoint shards: 100%|████████████████████████████████████████████████████████████████████████████████| 8/8 [00:11<00:00,  1.43s/it]
Some weights of ChatGLMForConditionalGeneration were not initialized from the model checkpoint at /root/autodl-tmp/chatglm-6b and are newly initialized: ['transformer.prefix_encoder.embedding.weight']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
INFO:     Started server process [335949]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
The dtype of attention mask (torch.int64) is not bool
[2023-05-08 17:52:57] ", prompt:"你知道清华不", response:"'采用<UNK>的面料，手感柔软，亲肤舒适，带来细腻的穿着体验。配合简约的款型，简洁大方，凸显品牌气质。搭配气质的衣袖，更显时尚。'"
INFO:     124.156.108.193:0 - "POST / HTTP/1.1" 200 OK
[2023-05-08 17:53:08] ", prompt:"你好", response:"'设计师将<UNK>作为主题，将<UNK>作为设计灵感，将<UNK>作为主题，将<UNK>作为主题，将<UNK>作为主题，将<UNK>作为主题，将<UNK>作为主题，将<UNK>作为主题，将<UNK'"
INFO:     124.156.108.193:0 - "POST / HTTP/1.1" 200 OK

**Comment by 488283943** [comment]
我想问一下putning后的权重放在哪个位置 怎么启用
